### DISCUSSION TITLE
Relation to X12 EDI Service Type Codes?

### DISCUSSION THREAD

**Discussion by nradov** [CMSgov/price-transparency-guide#745] [General]
Is there any interest in expanding the price transparency schema to include X12 EDI [Service Type Codes](https://x12.org/codes/service-type-codes)? Generally payers categorize billing codes into service type codes. But when doing EDI 270/271 eligibility checks with payers it isn't always clear which CPT / HCPCS codes they include under a particular Service Type Code (`EQ01`). It would be helpful for providers to close the loop on this if they could see Service Type Codes in the health plan MRFs.

**Comment by shaselton-usds** [comment]
Hi @nradov - We didn't receive any feedback that items/services are contracted under the X12 service type codes between the providers and payers. Is this something you've seen?

**Comment by nradov** [comment]
> Hi @nradov - We didn't receive any feedback that items/services are contracted under the X12 service type codes between the providers and payers. Is this something you've seen?

It's not a matter of how rates are contracted between providers and payers but rather helping providers understand how payers categorize particular billing codes. When a provider submits an X12 270 eligibility check the payer will return a 271 response with one or more service type codes. But the provider has no reliable way to know which benefit line applies to a particular billing code. Does that make sense?

**Reply by shaselton-usds** [reply]
Definitely makes sense. Generally speaking, the TiC requirements are for the contracted rates/terms and it sounds like those terms don't include X12 service type codes in the actual contracts.
